example/akamai.jsonnet

```diff
@@ -47,6 +47,24 @@ test.suite {
       ],
     },
 
+    // This test shows how to add a delay before the execution of a test
+    // This can be useful, if previous tests or actions require a delay before the next the can be executed
+    test.case {
+      name: 'Delay of test execution by 3 seconds',
+      pre:: [test.utils.delay(3000)], // Add a delay of 3s before the test is executed
```

Comment:
please simply use the flag

Reply:
I just created a new commit. Please check again.